arrow-syntax.api

```diff
@@ -0,0 +1,1632 @@
+	public static final fun uncurriedEffect (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function7;
+	public static final fun uncurriedEffect (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function8;
+	public static final fun uncurriedEffect (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function9;
+}
+
+public final class arrow/syntax/function/MemoizationKt {
+	public static final fun memoize (Lkotlin/jvm/functions/Function0;)Lkotlin/jvm/functions/Function0;
+	public static final fun memoize (Lkotlin/jvm/functions/Function10;)Lkotlin/jvm/functions/Function10;
+	public static final fun memoize (Lkotlin/jvm/functions/Function11;)Lkotlin/jvm/functions/Function11;
+	public static final fun memoize (Lkotlin/jvm/functions/Function12;)Lkotlin/jvm/functions/Function12;
+	public static final fun memoize (Lkotlin/jvm/functions/Function13;)Lkotlin/jvm/functions/Function13;
+	public static final fun memoize (Lkotlin/jvm/functions/Function14;)Lkotlin/jvm/functions/Function14;
+	public static final fun memoize (Lkotlin/jvm/functions/Function15;)Lkotlin/jvm/functions/Function15;
+	public static final fun memoize (Lkotlin/jvm/functions/Function16;)Lkotlin/jvm/functions/Function16;
+	public static final fun memoize (Lkotlin/jvm/functions/Function17;)Lkotlin/jvm/functions/Function17;
+	public static final fun memoize (Lkotlin/jvm/functions/Function18;)Lkotlin/jvm/functions/Function18;
+	public static final fun memoize (Lkotlin/jvm/functions/Function19;)Lkotlin/jvm/functions/Function19;
+	public static final fun memoize (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function1;
+	public static final fun memoize (Lkotlin/jvm/functions/Function20;)Lkotlin/jvm/functions/Function20;
+	public static final fun memoize (Lkotlin/jvm/functions/Function21;)Lkotlin/jvm/functions/Function21;
+	public static final fun memoize (Lkotlin/jvm/functions/Function22;)Lkotlin/jvm/functions/Function22;
+	public static final fun memoize (Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
+	public static final fun memoize (Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
+	public static final fun memoize (Lkotlin/jvm/functions/Function4;)Lkotlin/jvm/functions/Function4;
+	public static final fun memoize (Lkotlin/jvm/functions/Function5;)Lkotlin/jvm/functions/Function5;
+	public static final fun memoize (Lkotlin/jvm/functions/Function6;)Lkotlin/jvm/functions/Function6;
+	public static final fun memoize (Lkotlin/jvm/functions/Function7;)Lkotlin/jvm/functions/Function7;
+	public static final fun memoize (Lkotlin/jvm/functions/Function8;)Lkotlin/jvm/functions/Function8;
+	public static final fun memoize (Lkotlin/jvm/functions/Function9;)Lkotlin/jvm/functions/Function9;
+}
+
+public final class arrow/syntax/function/OptionKt {
+	public static final fun optionLift (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function1;
+}
+
+public final class arrow/syntax/function/PairingKt {
```

Comment:
Deprecate. Not useful since arity-n functions, and tupling is not supported in Kotlin.